opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/StatsPresenter.java

```diff
@@ -0,0 +1,15 @@
+package org.smartregister.reveal.presenter;
+
+import org.smartregister.reveal.contract.StatsContract;
+import org.smartregister.reveal.interactor.StatsInteractor;
+
+public class StatsPresenter implements StatsContract.Presenter {
+    private StatsContract.Interactor interactor;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'interactor' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4587928)